kubernetes/build.gradle

```diff
@@ -45,9 +45,7 @@ task createPythonDockerfile(type: Dockerfile) {
   copyFile("h2o_pysparkling_${sparkMajorVersion}-${version}.zip", "/opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip")
   runCommand """\\
                 pip install /opt/spark/pyspark/python/lib/h2o_pysparkling_${sparkMajorVersion}-${version}.zip && \\
-                apk add build-base python-dev && \\
```

Comment:
Don't you want to add a condition that `apk add` && `apk del` will be called up to the version 2.4.4?

Reply:
I was thinking about it and I think it makes sense to support images just for latest spark